scripts/check_codebase_language.py

```
@@ -0,0 +1,177 @@
+#!/usr/bin/env python3
```

Ruff (0.12.2)

Shebang is present but file is not executable

(EXE001)